.github/workflows/kind-e2e.yaml

```diff
@@ -32,7 +32,6 @@ jobs:
       fail-fast: false # Keep running if one leg fails.
       matrix:
         pipelines-release:
-        - v0.41.3  # LTS
```

Comment:
Wondering if we need this change in this PR? 🤔 
If so it seems that the v0.50.x might need an update following https://github.com/tektoncd/pipeline/blob/main/releases.md.

Reply:
My understanding is that we do need to change this in this PR.  In migrating from v1beta1 -> v1, Tekton v0.41.3 does not have the conversion webhook and thusly will not serve any Tekton Objects to the chains controller when the chains controller is watching v1.  This means that the tests would fail if they are still registered.  This was discussed in the Chains WG and it was stated this is ok to remove as it has rolled out of LTS support IIUC.

Reply:
Thanks @aaron-prindle this makes sense!
Just wondering in this case shall we add v0.53.2 as the latest LTS that has the resources conversion function brought back in this matrix?
cc @chitrangpatel

Reply:
Yes, but I don't think it needs to be a part of this PR.